teku/src/main/java/tech/pegasys/teku/cli/subcommand/VoluntaryExitCommand.java

```diff
@@ -257,10 +292,19 @@ private void initialise() {
             new PublicKeyLoader(),
             asyncRunner,
             metricsSystem,
-            Optional.empty());
+            dataDirLayout);
 
     try {
       validatorLoader.loadValidators();
+      pubKeysToExit.ifPresent(
```

Comment:
you could name the variable `maybePubKeysToExit` so the inner will be `pubKeysToExit`